pkg/clients/gitlab/git.go

```diff
@@ -120,3 +121,32 @@ func (gc *GitlabClient) CloseMergeRequest(projectID string, mergeRequestIID int)
 
 	return nil
 }
+
+// DeleteWebhooks deletes webhooks in Gitlab repo by given project ID
+func (gc *GitlabClient) DeleteWebhooks(projectID, consoleURL string) error {
+
+	// Check if "apps" is in consoleURL
+	appsIndex := strings.Index(consoleURL, "apps")
+	if appsIndex == -1 {
+		return fmt.Errorf("the string 'apps' was not found in the consoleURL: %s", consoleURL)
+	}
+
+	parsedConsoleURL := consoleURL[appsIndex:]
```

Comment:
Why do we need this step of removing app name?

Reply:
this how I get the sub string that should be same in both consoleURL and WebhookURL, see below example 

```
 consoleURL := "console-openshift-console.apps.93c0c83b29f76e50e138.hypershift.aws-2.ci.openshift.org"
    webhookURL := "https://pipelines-as-code-controller-openshift-pipelines.apps.93c0c83b29f76e50e138.hypershift.aws-2.ci.openshift.org"
```

Reply:
Maybe it's worth documenting

`parsedConsoleURL` should be probably named `appsURL` as nothing was really parsed here.

Reply:
why can't we just directly pass [f.ClusterAppDomain](https://github.com/konflux-ci/e2e-tests/blob/b58e6395af6829b78c3a72f4b1b338b7a0986417/pkg/framework/framework.go#L145) to this function and use it directly instead?

Reply:
thanks @dheerajodha , using `f.ClusterAppDomain` was the correct way, 
I've fixed the method